### ISSUE TITLE
genomics workflow template does not mount /dev/xvdba on /scratch

### ISSUE THREAD

**Issue by snamburi3** [aws-samples/aws-genomics-workflows#112]
The mount  /dev/xvdba [is included](https://github.com/aws-samples/aws-genomics-workflows/blob/77eeb21dacae434b5befbe3bd3d7b5b07eda5853/src/templates/gwfcore/gwfcore-launch-template.template.yaml#L87), but is not mounted at /scratch (or not used) on the instance. The /opt/ecs-additions/get-amazon-ebs-autoscale.sh proceeds to install  /dev/xvdbb directly.

The setup looks something like this below - with xvdba missing. I think this is a bug. Please let me know if I misunderstood. 

```
df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        3.9G   88K  3.9G   1% /dev
tmpfs           3.9G     0  3.9G   0% /dev/shm
/dev/xvda1      9.8G  1.6G  8.2G  16% /
/dev/xvdbb       100G   20G   31G  40% /var/lib/docker
```
quick workaround:
Instead of default /scratch, you can mount /dev/xvdba at /var/lib/docker by adding this in the runcmd here [on line](https://github.com/aws-samples/aws-genomics-workflows/blob/77eeb21dacae434b5befbe3bd3d7b5b07eda5853/src/templates/gwfcore/gwfcore-launch-template.template.yaml#L125):

```
- sed -i 's/install.sh/install.sh  -d \/dev\/xvdba /g' /opt/ecs-additions/get-amazon-ebs-autoscale.sh
```
the above hack will install /dev/xvdba at /var/lib/docker

**Comment by wleepang** [comment]
Probably best to add it directly to this line:
https://github.com/aws-samples/aws-genomics-workflows/blob/a700891d56b895f04d16cd0cf21675b4bed9f66e/src/ebs-autoscale/get-amazon-ebs-autoscale.sh#L164

**Comment by wleepang** [comment]
resolved in commit: b95b75d
